### PR TITLE
Add Synthetic Data and other

### PR DESCRIPTION
**Pull request by ib31iat**
- I added some information on synthetic data generation
- In the logic sectionI added: $\forall x: T(\phi)(x) = 0 \Leftrightarrow x\ \text{satisfies}\ \phi$.
- In the privacy section I added that **that shadow models** can be used for membership inference
- I created the chapter micellaneous to add some information.

Feel free to adapt / remove some changes.

### CODE REVIEW

File: chapters/miscellaneous.tex [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+\begin{itemize}
+  \item Hölder: $\|fg\|_1\leq\|f\|_p\cdot\|g\|_q$ ($\frac{1}{p}+\frac{1}{q}=1$)
+  \item Laplace: $f(x)=\frac{1}{2\sigma}e^{-\frac{|x-\mu|}{\sigma}}$
+  \item Normal dist: $f(x)=\frac{1}{2\sigma}e^{-\frac{1}{2}(\frac{|x-\mu|}{\sigma})^2}$
+  \item $x\Rightarrow y$ is equivalent to $\neg x \lor y$
+  \item DP: noise $\oplus$ $\rightarrow$ privacy $\oplus$ \& utility $\ominus$
+  \item Carlini-Wagner: $\|.\|_{\infty}\rightarrow\sum_i \max(\eta_i-\tau,0)$
+  \item Adversarial Accuracy: $\frac{\#\text{correct} - \#\text {adv\_examples}}{\#\text{samples}}$
+  \item Cross entropy loss: $-\sum_o\sum_c y_{o,c}log(p_{o,c})$
+\end{itemize}
```
Comment (hehli) [inline chapters/miscellaneous.tex:11]: We used $o$ to denote logits in the course so best not to use it as indices into a dataset.

Comment (ib31iat) [reply chapters/miscellaneous.tex:11]: True, I agree. Maybe $i$ makes sense? Or $s$ for sample?
